lib/main_common.pm

```diff
@@ -2692,26 +2692,26 @@ sub load_hypervisor_tests {
 
     if (check_var('VIRT_PART', 'snapshots')) {
         loadtest "virt_autotest/login_console";
-        loadtest "virtualization/xen/list_guests";
+        loadtest "virtualization/universal/list_guests";
 
         loadtest "virt_autotest/virsh_internal_snapshot";
         loadtest "virt_autotest/virsh_external_snapshot";
     }
 
     if (check_var('VIRT_PART', 'storage')) {
         loadtest "virt_autotest/login_console";
-        loadtest "virtualization/xen/list_guests";    # List all guests and ensure they are running
+        loadtest "virtualization/universal/list_guests";    # List all guests and ensure they are running
 
-        loadtest 'virtualization/xen/storage';        # Storage pool / volume test
+        loadtest 'virtualization/universal/storage';        # Storage pool / volume test
```

Comment:
Maybe you should grep more globally with "virtualization/xen", at least we have loaded xen/storage.pm and xen/hotplugging.pm tests in product/sle/main.pm, on lines https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/8b1940607749a07e71a84c8a0b6d5785540735c4/products/sle/main.pm#L788, https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/8b1940607749a07e71a84c8a0b6d5785540735c4/products/sle/main.pm#L789.

Besides can you help to add one verification for gi-guest_developing-on-host_developing-kvm( https://openqa.nue.suse.com/tests/4443361) so that we can know it does not fail our tests?

Reply:
Hello Alice, the [verification run](https://openqa.suse.de/tests/4550792#step/guest_installation_run/46) is failing but I'm not sure whether it's fail of this PR or not. Can you please take a look? Thank you

Reply:
HI Pavel, I don't know why it fails but it should not be caused by your change. you can rerun your test with UPLOAD_GUEST_ASSETS=0 to get it around, then the test will skip uploading guest assets and continue.

Reply:
I'm struggling with
```
Failed to run command 'nmap': No such file or directory
```
but I'm not avare that I changed something regarding `nmap`.
You can see the [gi-guest_developing-on-host_developing-kvm](https://openqa.nue.suse.com/tests/4559017#step/virsh_external_snapshot/44).

Reply:
OK, I introduced:
```perl
sub install_default_packages {
    # Install nmap, ip, dig
    zypper_call '-t in nmap iputils bind-utils', exitcode => [0, 4, 102, 103, 106];
}
```
which I'm calling at:
 * `tests/virt_autotest/install_package.pm`
 * `tests/virtualization/universal/ssh_hypervisor_init.pm`

Reply:
Finished green, check PR description. From my side it's ready for merge.